fe/fe-core/src/main/java/com/starrocks/persist/ColumnIdExpr.java

```diff
@@ -137,6 +137,9 @@ public String visitSlot(SlotRef node, Void context) {
             if (node.getTblNameWithoutAnalyzed() != null) {
                 return node.getTblNameWithoutAnalyzed().toString() + "." + node.getColumnId().getId();
```

Comment:
table name and column id also need back quoted?

Reply:
tks review.
I don't think it's necessary to add backquotes to tablet.columnId.
In SQL syntax parsing, if use a reserved keyword as an identifier, then user must add backquotes to it, but tablet.columnId is not a reserved keyword.

Reply:
Yes, you are right.
The tosql semantics should be guaranteed here